product_brand/product_brand_view.xml

```diff
@@ -71,25 +71,13 @@
             </field>
         </record>
 
-        <record model="ir.ui.view" id="normal_form_product_brand">
-           <field name="name">product.normal.form</field>
-           <field name="model">product.product</field>
-           <field name="inherit_id" ref="product.product_normal_form_view" />
-           <field name="arch" type="xml">
-                <div name="options" position="before">
-                    <label for="product_brand_id" class="oe_edit_only"/>
-                    <h2><field name="product_brand_id"/></h2>
-               </div>
-           </field>
-        </record>
-
-        <record model="ir.ui.view" id="product_template_form_brand_add">
+         <record model="ir.ui.view" id="product_template_form_brand_add">
             <field name="name">product.template.product.form</field>
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_form_view" />
             <field name="arch" type="xml">
-                <field name="categ_id" position="after"  >
-                  <field name="product_brand_id" />
+                <field name="name" position="after"  >
+                  <field name="product_brand_id" placeholder="Marque"/>
```

Comment:
placeholder shouldn't be in French, it could be "Brand" + a French translation in a po file eventually.